indico/modules/events/registration/templates/management/regform_edit.html

```diff
@@ -29,4 +29,43 @@
             {% trans %}Back{% endtrans %}
         </a>
     {% endcall %}
+
+    {% if regform %}
+        <script>
+            (function () {
+                'use strict';
+
+                $('#message-pending-preview').attr('data-href', '{{ url_for('.notification_preview', regform) }}').ajaxDialog({
+                    title: $T('Pending Registration Preview'),
+                    method: 'POST',
+                    data: function () {
```

Comment:
Arrow functions are a pretty cool things... :)

`data: () => ({state: ..., message:...});`

Reply:
Yep, indeed. Just copying the style of similar (and presumably old) files. But it's now fixed.